src/content/app/species-selector/state/species-selector-general-slice/speciesSelectorGeneralSlice.ts

```diff
@@ -37,13 +37,15 @@ import {
   getCommittedSpecies,
   getCommittedSpeciesById
 } from './speciesSelectorGeneralSelectors';
+import { LozengeOptionValues } from 'src/content/app/species-selector/components/species-lozenge-display-selector/SpeciesLozengeDisplaySelector';
```

Comment:
You've got a circular dependency there. `SpeciesLozengeDisplaySelector` imports stuff from `speciesSelectorGeneralSlice`, which tries to import something from `SpeciesLozengeDisplaySelector`.

Refer to my comments for `SpeciesLozengeDisplaySelector` for refactoring suggestions.